week1/homework/src/server.js

```diff
@@ -1,20 +1,49 @@
+
 'use strict';
 
 const http = require('http');
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
 function createServer(port) {
   let state = 10;
 
-  const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+  const server = http.createServer((req, response) => {
+    function printData() {
```

Comment:
Great job!
Try moving the printData function outside the  createServer function

Reply:
Done!